<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecentDock</class>
 <widget class="QDockWidget" name="RecentDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>327</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>164</width>
    <height>263</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset theme="document-open-recent">
    <normaloff>:/icons/oxygen/32x32/actions/document-open-recent.png</normaloff>:/icons/oxygen/32x32/actions/document-open-recent.png</iconset>
  </property>
  <property name="windowTitle">
   <string>Recent</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="LineEditClear" name="lineEdit">
        <property name="toolTip">
         <string>Show only files with name matching text</string>
        </property>
        <property name="placeholderText">
         <string>search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>469</width>
         <height>263</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected item</string>
   </property>
  </action>
  <action name="actionRemoveAll">
   <property name="text">
    <string>Remove All</string>
   </property>
   <property name="toolTip">
    <string>Remove all items</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play the item</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
   <property name="toolTip">
    <string>Show properties</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditClear</class>
   <extends>QLineEdit</extends>
   <header>widgets/lineeditclear.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
